bashbrew/go/src/bashbrew/cmd-put-shared.go

```diff
@@ -48,9 +52,6 @@ func entriesToManifestToolYaml(r Repo, entries ...*manifest.Manifest2822Entry) (
 			}
 		}
 	}
-	if yaml == "" {
-		return "", time.Time{}, fmt.Errorf("failed gathering images for creating %q", entryIdentifiers)
-	}
```

Comment:
I did verify -- `manifest-tool` and the registry are both happy to accept an empty manifest list. :+1:

Reply:
(This is necessary for cases like `arm32v6/hello-world:latest` which doesn't exist but is still a `SharedTag` we supposedly need to try pushing.)